agent/s3/s3.go

```diff
@@ -50,3 +52,24 @@ func ParseS3ARN(s3ARN string) (bucket string, key string, err error) {
 	}
 	return match[2], match[3], nil
 }
+
+func GetObject(bucket string, key string, client s3iface.S3API) (string, error) {
+	requestInput := &s3.GetObjectInput{
+		Bucket: aws.String(bucket),
+		Key:    aws.String(key),
+	}
+
+	result, err := client.GetObject(requestInput)
+	if err != nil {
+		return "", err
+	}
+
+	defer result.Body.Close()
+	resultBody, err := ioutil.ReadAll(result.Body)
```

Comment:
looks like ioutil is deprecated: https://pkg.go.dev/io/ioutil

you can use `io.ReadAll(...)` instead.

Reply:
sure